### DISCUSSION TITLE
Clickable element inside tab ?

### DISCUSSION THREAD

**Discussion by KaKi87** [jarek-foksa/xel#120] [Q&A]
Hello,

I would like to make a closable `<x-tab>`, so I'm adding a *close* button inside it.
But, the `click` event from it is never fired (only the one from the tab itself).

- [Code snippet 1 : using `xel`](https://jsbin.com/vabijevifo/edit?html,css,js,output)
- [Code snippet 2 : not using `xel`](https://jsbin.com/kadidaxote/edit?html,css,js,output)

Thanks

**Comment by jarek-foksa** [comment]
If you need the tabs to be closable and draggable, you should probably implement your own custom widget type (you could reuse the code from the deprecated [`<x-doctabs>`](https://github.com/jarek-foksa/xel/commit/7db012c5a2126ff9e6bcab38ccf2a0852aa6f015#diff-d4019a44b0b5a85454f4c26ee5c3a784571c1d02deb28a04094114b0310d0e56)).

I'm not really sure why the click event is not fired in your example, but it is most likely because you did put an `<a>` element inside it.
